### ISSUE TITLE
https://quay.io/repository/containerdisks/centos-stream is consuming 187.44 GiB of quota within quay.

### ISSUE THREAD

**Issue by lyarwood** [kubevirt/containerdisks#101]
**What happened**:

https://quay.io/repository/containerdisks/centos-stream is consuming 187.44 GiB of quota within quay.

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:
Steps to reproduce the behavior.

**Additional context**:

**Environment**:
- KubeVirt version (use `virtctl version`): N/A
- Kubernetes version (use `kubectl version`): N/A
- VM or VMI specifications: N/A
- Cloud provider or hardware configuration: N/A
- OS (e.g. from /etc/os-release): N/A
- Kernel (e.g. `uname -a`): N/A
- Install tools: N/A
- Others: N/A

**Comment by 0xFelix** [comment]
How many recent versions are stored on the CentOS Stream mirror? Do they delete old versions at some point?

**Comment by lyarwood** [comment]
@jcanocan yes I think that would be more useful, I wouldn't even be against setting it to a low number across the project to start with and seeing what if any feedback we get from users is. I assume most are just using the `latest` or some other top level version alias like `fedora:42` etc

**Comment by jcanocan** [comment]
> [@jcanocan](https://github.com/jcanocan) yes I think that would be more useful, I wouldn't even be against setting it to a low number across the project to start with and seeing what if any feedback we get from users is. I assume most are just using the `latest` or some other top level version alias like `fedora:42` etc

What about a maximum of 100-150 tags? Repos with more than that are from 1 year or 4 months ago.

**Comment by 0xFelix** [comment]
I don't see settings for this in the containerdisks org on quay.io.

@brianmcarey @dhiller Is it even possible to change the above-mentioned quota settings on quay.io?

**Comment by dhiller** [comment]
According to the [org quota docs](https://docs.projectquay.io/api_quay.html#quota-management-api) there's currently only storage limits that you can set.

**Comment by dhiller** [comment]
These can only be changed in the API AFAICT - there's no UI inside the org settings for this

**Comment by codingben** [comment]
> Okay. I left a message in our #forum-quay Slack channel to get an answer if it can be enabled for us.

This Quay's auto-pruning feature is not enabled in public Quay.
